src/docs/index.rst

```diff
@@ -29,46 +29,31 @@ Serac is hosted on `GitHub <https://github.com/LLNL/serac>`_. Serac uses git sub
    # for some commands
    $ git clone --recursive https://github.com/LLNL/serac.git
 
-Preparing Windows WSL/Ubuntu for Serac installation
----------------------------------------------------------
-
-For faster installation of the Serac dependencies via Spack on Windows WSL/Ubuntu systems, install cmake, MPICH, openblas, OpenGL, and the devtools
-using the following commands for Ubuntu 20.04:
-
-.. code-block:: bash
+Overview of the Serac build process
+------------------------------------
 
-   $ sudo apt-get update
-   $ sudo apt-get upgrade
-   $ sudo apt-get install cmake libopenblas-dev libopenblas-base mpich mesa-common-dev libglu1-mesa-dev freeglut3-dev cppcheck doxygen libreadline-dev
-   $ sudo ln -s /usr/lib/x86_64-linux-gnu/* /usr/lib
-
-and the following commands for Ubuntu 18.04:
-
-.. code-block:: bash
+The serac build process has been broken into three phases with various related options:
 
-   $ sudo apt-get update
-   $ sudo apt-get upgrade
-   $ sudo apt-get install g++-8 gcc-8
-   $ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8
-   $ sudo apt-get install cmake libopenblas-dev libopenblas-base mpich mesa-common-dev libglu1-mesa-dev freeglut3-dev cppcheck doxygen libreadline-dev
-   $ sudo ln -s /usr/lib/x86_64-linux-gnu/* /usr/lib
+1. (Optional) Build the developer tools
+2. Build the third party libraries
+3. Build the serac source code
 
-Note that the last line is required since Spack expects the system libraries to exist in a directory named ``lib``. During the third
-party library build phase, the appropriate Spack config directory must be specified using either 
-``python scripts/uberenv/uberenv.py --spack-config-dir=scripts/uberenv/spack_configs/linux_ubuntu_18`` or
-``python scripts/uberenv/uberenv.py --spack-config-dir=scripts/uberenv/spack_configs/linux_ubuntu_20`` as appropriate.
+The developer tools are only required if you wish to contribute to the Serac source code. The first two steps involve building all of the 
+third party libraries that are required by Serac. Two options exist for this process: using the `Spack HPC package manager <https://spack.io/>`_
+via the `uberenv wrapper script <https://github.com/LLNL/uberenv>`_ or building the required dependencies on your own. We recommend the first
+option as building HPC libraries by hand can be a tedious process. Once the third party libraries are built, Serac can be built using the
+cmake-based `BLT HPC build system <https://github.com/LLNL/blt>`_.
 
 Building Serac's Developer Tools
 --------------------------------
 
 .. note::
   This can be skipped if you are not doing Serac development or if you are on an LC machine.
-  We have them installed for you in a public space defined in 
-  ``host-config/<machine name>-<SYS_TYPE>-<compiler>.cmake``
+  They are installed in a public space defined in ``host-config/<machine name>-<SYS_TYPE>-<compiler>.cmake``
```

Comment:
Note these aren't exactly "public" they are guarded by our group.  We could on the otherhand, use Axom's devtools which are built in /collab/usr/gapps/axom and open to most of anyone that has LC access.